docs/even-segmented-path-for-put-operation.md

```diff
@@ -0,0 +1,33 @@
+# EvenSegmentedPathForPutOperation
+
+## Category
+
+ARM Error
+
+## Applies to
+
+ARM OpenAPI(swagger) specs
+
+## Related ARM Guideline Code
+
+- RPC-Put-V1-02
+
+## Output Message
+
+API path with PUT operation defined MUST have even number of segments (i.e. end in {resourceType}/{resourceName} segments).
+
+## Description
+
+API path with PUT operation defined MUST have even number of segments (i.e. end in {resourceType}/{resourceName} segments).
+
+## How to fix the violation
+
+Fix the path with PUT defined as one of the below patterns:
+
```

Comment:
Its not clear if u are calling the below paths as the problem or the right way to specify the paths

Reply:
can u rephrase the sentence to reflect the inent